.travis.yml

```diff
@@ -19,7 +19,7 @@ env:
   - LOGBACK_ROOT_LEVEL=WARN
   - LOGBACK_EXCEPTION_PATTERN=%xThrowable{3}
 
-scala_version_212: &scala_version_212 2.12.9
+scala_version_212: &scala_version_212 2.12.8
```

Comment:
We were on 2.12.9 already. Now we've overshot. But I don't care enough to keep blocking this.